### DISCUSSION TITLE
Windows App SDK unpackaged sample do not run in admin mode.

### DISCUSSION THREAD

**Discussion by dimohy** [microsoft/WindowsAppSDK#2313] [Q&A] [answered]
If 'Unpackaged' of the Windows App SDK sample is run in administrator mode, error 0xc0000409 occurs in Microsoft.WindowsAppRuntime.Bootstrap.dll and does not work.

https://github.com/microsoft/WindowsAppSDK-Samples/tree/main/Samples/Unpackaged/cs-console-unpackaged

If you have successfully installed the Windows App SDK runtime and are not in admin mode, it will run.

In a special environment, it must be run in administrator mode.

I want to know why it is not running in admin mode and I want to run it in admin mode.

**Comment by dimohy** [comment]
For example, Windows Sandbox runs in administrator mode when running applications. Unable to run Windows App SDK unpackaged apps in Windows Sandbox.

**Reply by DominicMaas** [reply]
That saves me some time! I'm using Windows Sandbox to test an installer that installs all the WASDK dependencies (wasdk, .net6 runtime, vclibs) and was super confused why the app was not starting / wondering if the installer was not working). Looks like it's likely admin issues.

**Comment by dimohy** [comment, marked as answer]
Upon further checking, the program does not seem to work when privileges are elevated in version 1.0 (or 1.0.1), but according to the plan, it seems that it has been modified to work even when privileges are elevated in version 1.1.

https://portal.productboard.com/winappsdk/1-windows-app-sdk/c/37-elevated-admin-support

https://github.com/microsoft/WindowsAppSDK/pull/2066

Looking at the pull request in this regard, it appears that the request is ready. Is there a way to pre-use in version 1.0.1?

**Reply by DrusTheAxe** [reply]
1.0 doesn't support elevation, for multiple reasons.

1.1 supports elevation.

**Reply by dimohy** [reply]
Thanks for your reply.

I am also aware that this behavior was deprecated in 1.0.

What I'm looking for is whether there is a way to fix the problem now with a bootstrap-related hack or as a separate suggestion from 1.0.

**Reply by DrusTheAxe** [reply]
> I am also aware that this behavior was deprecated in 1.0.

Ohhh it wasn't deprecated. It was flat out not supported.

Unpackaged apps weren't supported until 1.0, and not elevated for multiple reasons (it's not just the bootstrapper). We've spent a healthy amount of time addressing those blockers. I can't think of a way one could fiddle with 1.0 to make it work. Sorry.

**Reply by dimohy** [reply]
Thanks for helping me understand the status exactly.
So, when will I get a version with elevated privileges? It's ok even if it's a preview version.

**Reply by DrusTheAxe** [reply]
>when will I get a version with elevated privileges? It's ok even if it's a preview version.

The updates are in main and testing's underway. I don't know dates (and even if I did I probably couldn't say) but...RSN :-)

**Reply by dimohy** [reply]
Thanks for your reply. :) T_T
